src/App.js

```diff
@@ -39,12 +39,11 @@ class App extends Component {
           <List
             tab={tasks}
             classListName={'tasksList'}
-            classListItemName={'task'}
+            classListItemName={'tasksList__item'}
```

Comment:
it would be nice, if you could manage during this code review nicer convention for us, so not use camelCase for css classes but kebab case (in example: `task-list__item`). 
this is only the convention, but it really improve readability in the final codebase ;)